templates/shortcode-receipt.php

```diff
@@ -148,7 +148,7 @@
 						?>
 
 						<div class="edd_purchase_receipt_product_name">
-							<?php echo esc_html( $item['name'] ); ?>
+							<?php echo esc_html( edd_get_cart_item_name( $item ) ); ?>
 							<?php if( ! is_null( $price_id ) ) : ?>
 							<span class="edd_purchase_receipt_price_name">&nbsp;&ndash;&nbsp;<?php echo edd_get_price_option_name( $item['id'], $price_id, $payment->ID ); ?></span>
```

Comment:
@pippinsplugins This would be okay.
```php
<div class="edd_purchase_receipt_product_name">
	<?php
	if( empty( $item['name'] ) ):
		echo $item['id'];
	else:
		echo esc_html( $item['name'] );
	endif;
	?>
	<?php if( ! is_null( $price_id ) ) : ?>
	<span class="edd_purchase_receipt_price_name">&nbsp;&ndash;&nbsp;<?php echo edd_get_price_option_name( $item['id'], $price_id, $payment->ID ); ?></span>
	<?php endif; ?>
</div>
```